.github/workflows/audit.yml

```diff
@@ -0,0 +1,24 @@
+name: Audit
+
+on:
+  pull_request:
+    branches:
+      - master
+  # schedule:
+  #   - cron: '0 10 * * *'
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x

```

Comment:
For workflows like this one that don't run on a matrix of multiple Node versions, should we consider specifying [`node-version: lts/*`](https://github.com/actions/setup-node#supported-version-syntax) so we don't have to update the number manually in the future? 😛

Reply:
I like that suggestion!
Adding it now 😋

Reply:
Messed up the first time, but updated the setup-node version to v2 and now it works just fine! 😋